webpack/library.ts

```diff
@@ -0,0 +1,116 @@
+/**
```

Comment:
More explanatory name than "library" would serve well? `build-of-extensions.ts`? Also it would be better to make the extensions responsible for getting built, if I'm not missing something.